.github/workflows/ci.yml

```diff
@@ -32,11 +32,6 @@ jobs:
             os: ubuntu-latest
             toxenv: py38
 
-          - name: Python 3.7 Schema validation tests
-            python-version: 3.7
-            os: ubuntu-latest
-            toxenv: py37
-
```

Comment:
is this test no longer necessary?

Reply:
I was getting odd errors with astropy 4, which we do not support  anymore.